/adas/camera_ros_node:
  ros__parameters:
    url: rtsp://admin:bdtd123456@192.168.1.36:554/h265/ch1/main/av_stream    # 0, rtsp_url,etc
    output_h: 640
    output_w: 640
    image_pub_topic_name: /adas/video_frames
    log_filedir_prefix: /home/nvidia/datalog/adas/system_log/
    rect_x: 0
    rect_y: 0
    rect_h: 1080
    rect_w: 1920                      # rect represent image
